pygmt/clib/conversion.py

```diff
@@ -268,9 +226,9 @@ def kwargs_to_ctypes_array(argument, kwargs, dtype):
     --------
 
     >>> import ctypes as ct
-    >>> value = kwargs_to_ctypes_array('bla', {'bla': [10, 10]}, ct.c_int*2)
+    >>> value = kwargs_to_ctypes_array('bla', {'bla': [10, 10]}, ct.c_long*2)
     >>> type(value)
-    <class 'pygmt.clib.conversion.c_int_Array_2'>
+    <class 'pygmt.clib.conversion.c_long_Array_2'>
```

Comment:
`c_int` is just an alias for `c_long` according to https://docs.python.org/3/library/ctypes.html#ctypes-tutorial. Changing it to `c_long` should make this test pass in a cross-platform way.